lib/keychain/keychain.rb

```diff
@@ -102,6 +136,37 @@ def generic_passwords
       Scope.new(Sec::Classes::GENERIC, self)
     end
 
+    # Imports item from string or file to this keychain
+    #
+    # @param [String, File] input String or File with raw data to import
+    # @param [Array <String>] trusted_apps List of applications which will be
+    # permitted to access imported items
+    def import(input, trusted_apps=[])
```

Comment:
>Would it be useful if this returned the imported item?
I've thought about it, but have not find a good way to do this. We can fetch the resulted object by setting the pointer to the last argument of `SecItemImport `, but I don't know what to do with it next. This object has empty `@attributes` dict and I can't call load attributes there, because this object doest not linked to come keychain. Huh... I'm stucked

> Should you accept IO rather than file (thinking of people piping in things) ?
Yes, I think it's reasonable. Will do it soon